packages/manager/apps/pci-databases-analytics/src/hooks/api/database/service/useEditService.hook.tsx

```diff
@@ -8,10 +9,18 @@ interface UseEditService {
   onSuccess: (service: database.Service) => void;
 }
 export function useEditService({ onError, onSuccess }: UseEditService) {
+  const queryClient = useQueryClient();
+  const { projectId } = useParams();
   const mutation = useMutation({
     mutationFn: (serviceUpdate: EditService) => editService(serviceUpdate),
     onError,
-    onSuccess,
+    onSuccess: (data: database.Service) => {
+      onSuccess(data);
```

Comment:
There is a misunderstanding between the two "onSuccess" functions. It 
will be better to change the name of the function and avoid both functions to have the same naming.

Reply:
Yes, I agree. Fixed